### PR TITLE
Add fixtures for seo_location table

### PR DESCRIPTION
**Pull request by sareensumanau**

### CODE REVIEW

File: latest-fixtures.sql [hunk 1]
```diff
@@ -15,3 +15,10 @@ INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('
 INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('10@example.com', '25LAw8Yzc5SxsMi5DlHIgMoNbVL', '2021-01-01', '2021-01-01');
 INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('11@example.com', '25LAw8Yzc5SxsMi5DlHIgMoNbVM', '2021-01-01', '2021-01-01');
 INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('12@example.com', '25LAw8Yzc5SxsMi5DlHIgMoNbVN', '2021-01-01', '2021-01-01');
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('United States', '$', 'United States', ':-)');
```
Comment (dm161) [inline latest-fixtures.sql:18]:
```suggestion
INSERT INTO seo_location (name, currency, country, emoji) VALUES ('United States', '$', 'United States', '🇺🇸');
```

File: latest-fixtures.sql [hunk 2]
```diff
@@ -15,3 +15,10 @@ INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('
...
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('United Kingdom', '£', 'United Kingdom', ':-)');
```
Comment (dm161) [inline latest-fixtures.sql:19]:
```suggestion
INSERT INTO seo_location (name, currency, country, emoji) VALUES ('United Kingdom', '£', 'United Kingdom', '🇬🇧');
```

File: latest-fixtures.sql [hunk 3]
```diff
@@ -15,3 +15,10 @@ INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('
...
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('Germany', '€', 'Germany', ':-)');
```
Comment (dm161) [inline latest-fixtures.sql:20]:
```suggestion
INSERT INTO seo_location (name, currency, country, emoji) VALUES ('Germany', '€', 'Germany', '🇩🇪');
```

File: latest-fixtures.sql [hunk 4]
```diff
@@ -15,3 +15,10 @@ INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('
...
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('China', '¥', 'China', ':-)');
```
Comment (dm161) [inline latest-fixtures.sql:21]:
```suggestion
INSERT INTO seo_location (name, currency, country, emoji) VALUES ('China', '¥', 'China', '🇨🇳');
```

File: latest-fixtures.sql [hunk 5]
```diff
@@ -15,3 +15,10 @@ INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('
...
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('Japan', 'JP¥', 'Japan', ':-)');
```
Comment (dm161) [inline latest-fixtures.sql:22]:
```suggestion
INSERT INTO seo_location (name, currency, country, emoji) VALUES ('Japan', 'JP¥', 'Japan', '🇯🇵');
```

File: latest-fixtures.sql [hunk 6]
```diff
@@ -15,3 +15,10 @@ INSERT INTO email_subscribers (email, token, confirmed_at, created_at) VALUES ('
...
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('India', '₹', 'India', ':-)');
+INSERT INTO seo_location (name, currency, country, emoji) VALUES ('France', '€', 'France', ':-)');
```
Comment (dm161) [inline latest-fixtures.sql:24]:
```suggestion
INSERT INTO seo_location (name, currency, country, emoji) VALUES ('France', '€', 'France', '🇫🇷');
```

**Comment by sareensumanau** [comment]
Build is failing because of
```
Failed to resolve action download info. Error: Circuit Breaker "HttpClientThrottler-GitHubVssHttpClientRequester-launch-receiver.githubapp.com" short-circuited. In the last 10000 milliseconds, there were: 1 failure, 0 timeout, 91 short circuited, 0 concurrency rejected, and 0 limit rejected. Last exception: Microsoft.VisualStudio.Services.WebApi.VssServiceResponseException Internal Server Error
```

@dm161  Do you know what's causing this ?

**Comment by dm161** [comment]
Github action temporarily down try later

**Pull request merged**